### ISSUE TITLE
TS-Standard fails in Action but not locally

### ISSUE THREAD

**Issue by dominicmeyer** [super-linter/super-linter#4897]
## Describe the bug

Only when running super-linter this error exists, I fixed it when `ts-standard` locally.

```
Parsing error: ESLint was configured to run on <file> using `parserOptions.project`: <tsconfigRootDir>/../../action/lib/.automation/tsconfig.json
However, that TSConfig does not include this file.
```

I fixed it by adding a `tsconfig.eslint.json' file and including all files, that I want to lint.
I don't know if that's important, but right now I'm running super linter with `VALIDATE_ALL_CODEBASE: true`.

## Expected behavior

Running `ts-standard` should be the same locally and trough super-linter.

## Steps to Reproduce

A minimal reproduction can be found here: [https://github.com/dominicmeyer/ts-eslint-error](https://github.com/dominicmeyer/ts-eslint-error). \
The README should explain how to see the error.

**Comment by ferrarimarco** [comment]
Thanks for this report, and the effort to reproduce the issue.

This is probably due to the [default configuration not including your particular file](https://github.com/super-linter/super-linter/blob/df1420ce9dc7091aba910c3a416eb03d98fcc368/TEMPLATES/tsconfig.json), as stated in the error.
We include some defaults, tho, but this may not be enough, as you experienced.

Given how [TypeScript uses tsconfig.json](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html), does it even make sense to provide a default one, instead of asking the user to provide one?

Additionally, we might have issues about how we invoke `ts-standard`:

https://github.com/super-linter/super-linter/blob/df1420ce9dc7091aba910c3a416eb03d98fcc368/lib/linter.sh#L1006

- Is `${TYPESCRIPT_STANDARD_LINTER_RULES}` taken into consideration? [ts-standard docs](https://github.com/standard/ts-standard#-help) say that any file that's not prefixed by options, is a file that `ts-standard` should lint, and not a configuration file.
- Even if we forget about the previous point, is `TYPESCRIPT_STANDARD_LINTER_RULES` taken into account at all? It [currently defaults to `.eslintrc.yml`](https://github.com/super-linter/super-linter/blob/df1420ce9dc7091aba910c3a416eb03d98fcc368/lib/linter.sh#L218), and it seems that `ts-standard` doesn't support customizing it ([related issue](https://github.com/standard/ts-standard/issues/103), [related code](https://github.com/standard/ts-standard/blob/5e45a1e2780f720de1328bf15bf71255881ffa7f/options.js#L9)).

I think both these issues are probably due to leftovers from #2614 (/cc @massongit, the author for visibility).
